.env.test

```diff
@@ -6,3 +6,5 @@ WED_DB_USER=postgres
 WED_DB_PWD=The_Traub
 DEVISE_PEPPER=9431fb43f1cc76749ec8c3d6e044e5109637078a85daf99b173a6cf6a62198105c0f148a99876901fd124ee436044188d0594c4d467a07a4883d5476bfcd1892
 DEVISE_SECRET_KEY=abcdefghijklm123456789
+SIDEKIQ_USERNAME=async
```

Comment:
Note the lack of `REDIS_URL` here - this is intentional, as Sidekiq should never be hitting Redis during specs.